developer/src/kmc-copy/test/copier.tests.ts

```diff
@@ -523,6 +524,103 @@ describe('KeymanProjectCopier', function() {
+      'organized/source/organized.kps',
+      'organized/source/readme.htm',
+      // files in 'another-folder/' get moved to 'external/'
+      'organized/external/another-folder/welcome/welcome.htm',
+      'organized/external/another-folder/welcome/KeyboardLayout.png',
+      'organized/external/another-folder/welcome/KeyboardLayoutAlt.png',
+      'organized/external/another-folder/welcome/KeyboardLayoutShift.png',
+      // build artifact files get moved to build/
+      'organized/build/organized.js',
+      'organized/build/organized.kvk',
+      'organized/build/organized.kmx',
+      // 'organized/build/organized.kmp', // TODO-COPY: do we need to copy this file as well?
+      'organized/LICENSE.md',
+      'organized/organized.kpj',
+      'organized/HISTORY.md',
+      'organized/README.md',
+      'organized/source/help/organized.php',
+      'organized/source/help/KeyboardLayout.png',
+      'organized/source/help/KeyboardLayoutAlt.png',
+      'organized/source/help/KeyboardLayoutShift.png',
+    ]);
+
+    // Verify that .kps has the appropriate paths
+    const reader = new KpsFileReader(callbacks);
+    const kpsFilename = Object.keys(result.artifacts).find(k => result.artifacts[k].filename == 'organized/source/organized.kps')!;
+    const kps = reader.read(result.artifacts[kpsFilename].data);
+    assert.isDefined(kps.Package.Files?.File.find(file => file.Name == '../external/another-folder/welcome/welcome.htm'));
+    assert.isDefined(kps.Package.Files?.File.find(file => file.Name == '../external/another-folder/welcome/KeyboardLayout.png'));
+    assert.isDefined(kps.Package.Files?.File.find(file => file.Name == '../external/another-folder/welcome/KeyboardLayoutAlt.png'));
+    assert.isDefined(kps.Package.Files?.File.find(file => file.Name == '../external/another-folder/welcome/KeyboardLayoutShift.png'));
+  });
+
+  it('should leave external files in their original location when relocateExternalFiles is false', async function() {
+    const copier = new KeymanProjectCopier();
+    assert.isTrue(await copier.init(callbacks, {
+      dryRun: false,
+      outPath: 'organized',
+      relocateExternalFiles: false,
+    }));
+
+    const result = await copier.run(makePathToFixture('projects','external','alephwithbeth','alephwithbeth.kpj'));
+    assert.sameMembers(Object.keys(result.artifacts).map(k => result.artifacts[k].filename), [
+      'organized', // kmc-copy:outputPath
+      'organized/source/organized.keyman-touch-layout',
+      'organized/source/organized.kmn',
+      'organized/source/organized.ico',
+      'organized/source/organized.kvks',
+      'organized/source/organized.kps',
+      'organized/source/readme.htm',
+
+      // files in 'another-folder/' stay there, so they are not referenced
+      // 'organized/external/welcome.htm',
+      // 'organized/external/KeyboardLayout.png',
+      // 'organized/external/KeyboardLayoutAlt.png',
+      // 'organized/external/KeyboardLayoutShift.png',
```

Comment:
Does this mean the files aren't referenced in the original `another-folder/` location?

Reply:
No, it means that the project copier doesn't have artifacts to create these files, because they are already there.